yt/loaders.py

```diff
@@ -1298,9 +1301,15 @@ def load_sample(fn=None, progressbar: bool = True, timeout=None, **kwargs):
         `None` means "no limit". This parameter is directly passed to down to
         requests.get via pooch.HTTPDownloader
 
-    Any additional keyword argument is passed down to `yt.load`.
-    Note that in case of collision with predefined keyword arguments as set in
+    Notes
+    -----
+    - Any additional keyword argument is passed down to `yt.load`.
+    - In case of collision with predefined keyword arguments as set in
     the data registry, the ones passed to this function take priority.
+    - Datasets with slashes '/' in their names can safely be used even on Windows.
+      On the contrary, paths using backslashes '\' won't work outside of Windows, so
+      it is recommended to favour the UNIX convention ('/') in scripts that are meant
+      to be cross-platform.
```

Comment:
Would it be possible to include a short description of what `load_sample` does here in the docstring as well as what format the filenames should take.  Yes, they're strings, but for `IsolatedGalaxy` it seems like that is sufficient to load that dataset, whereas `Enzo_64` one needs to specify the specific dataset within that directory fully.  Maybe also have the URL of the actual location where the datasets are?  Or that what the behavior is when you specify no arguments? I could do this as a separate PR if need be.

Reply:
The specificity in the Enzo_64 tarball is that there's a whole time series in there. We could choose one specific snapshot that would load if nothing else is supplied. I don't want to add to much detail here because I don't want to let users think that the design is 100% intentional. I'll try to clarify a bit though.

Reply:
I intentionally avoided going too much into the details, but here's an attempt to satisfy your question !